CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 - Add `named_only` style to `RSpec/NamedSubject`. ([@kuahyeow])
 - Fix `RSpec/FactoryBot/ConsistentParenthesesStyle` to ignore calls without the first positional argument. ([@pirj])
+- Fix `RSpec/NestedGroups` to correctly use `AllowedGroups` config. ([@samrjenkins])
```

Comment:
To fix the Markdown check, you have to add yourself in a section listing contributors at the end of this file, maintaining the alphabetic order.

Reply:
This used to be pointed out by one of our changelog specs, but it seems to be not working since we change the Markdown format recently. I’ll go fix it right away.

Reply:
Ah! Thanks 🚀. I have addresses